spec/controllers/account/devices_controller_spec.rb

```diff
@@ -0,0 +1,113 @@
+require 'rails_helper'
+
+RSpec.describe Account::DevicesController, type: :controller do
+  render_views
+
+  describe 'GET #index' do
+    context 'with signed in user' do
+      let!(:user) { create(:user) }
+
+      before(:each) do
+        sign_in user
+        get :index
+      end
+
+      subject { response }
+
+      it { is_expected.to have_http_status(200) }
+      it { is_expected.to render_template('index') }
+    end
+  end
+
+  describe 'GET #new' do
+    let!(:user) { create(:user) }
+
+    context 'with signed in user' do
+      before(:each) do
+        sign_in user
+        get :new
+      end
+
+      subject { response }
+
+      it { is_expected.to have_http_status(200) }
+      it { is_expected.to render_template('new') }
+    end
+  end
+
+  describe 'GET #edit' do
+    let!(:user) { create(:user, own_organization: organization) }
+    let!(:organization) { create(:organization_with_device) }
+
+    context 'with signed in user' do
+      before(:each) do
+        sign_in user
+        get :edit, params: { id: user.own_organization.devices.first.id }
```

Comment:
where does this come from?
test should be explicit, to understand where this or that data come from and how it should behave